src/components/views/Order/PersonalDataForm/config.js

```diff
@@ -0,0 +1,75 @@
+const patterns = {
+  namePattern: /^[A-zĄ-ż ,.-]*$/,
+  streetPattern: /^[A-zĄ-ż0-9 ,.-/]*$/,
+  numberPattern: /^[A-zĄ-ż0-9 ,.-/]*$/,
+  phonePattern: /^[0-9 +-]*$/,
+};
+
+export const inputFields = [
+  {
+    name: 'firstName',
+    label: 'first name',
+    errorMsg: 'First name required, should be 2-20 ch., may include . , - special characters.',
```

Comment:
Skoro już w tym kierunku poszłaś to bym się zastanowił nad przeniesieniem wiadomości o błędach do osobnej metody i tylko wskazaniem ich tutaj. Na zasadzie:
const errorMessages = {
  required: '{field} is required
}

I potem w polu:
errorMsg = ['required', '...']

A gdy przyszłoby do wyświetlania, to podmieniałabyś wartości w {} na zmienne. Na zasadzie:
errorMessages.required.replace('{value}', field.name). Tym sposobem zamiast mieć konkretne wiadomości dla pól miałabyś wiadomości składane z kawałków.